Package.swift

```diff
@@ -120,9 +116,7 @@ var targets: [Target] = [
       .product(name: "Markdown", package: "swift-markdown"),
       .product(name: "SwiftOperators", package: "swift-syntax"),
       .product(name: "SwiftParser", package: "swift-syntax"),
```

Comment:
Should these two also be moved into `swiftSyntaxDependencies`?

Reply:
Indeed.